### ISSUE TITLE
https://www.virustotal.com/

### ISSUE THREAD

**Issue by er512** [Feodor2/Mypal#83]
Hello, page is loaded not correctly, Test API is not displayed.
Thx for fix it.

**Comment by andika207** [comment]
@er512 why do you open a new issue for every unworking site ?
it would be cool if you post all of your shit inside the same issue so it makes this repo clearer for @Feodor2 and you have a minimal respect for other users who track this repo ok ?

**Comment by win98se** [comment]
Agreed. If you are using Mypal68, there is a dedicated issue just for the non-working sites - https://github.com/Feodor2/Mypal68/issues/82

Just comment in the issue with all those non-working sites.
